climada/hazard/tc_tracks.py

```diff
@@ -1851,3 +1854,51 @@ def set_category(max_sus_wind, wind_unit='kn', saffir_scale=None):
         return (np.argwhere(max_wind < saffir_scale) - 1)[0][0]
     except IndexError:
         return -1
+
+def make_line_from_track(track, split_lines_antimeridian):
+    lon = u_coord.lon_normalize(track.lon.values.copy(), center=0.0)
+    lat = track.lat.values
+    # LineString only works with more than one lat/lon pair
+    if lon.size == 1:
+        return Point(lon, lat)
+    extent = u_coord.lon_bounds(lon)
+    if split_lines_antimeridian and extent[0] <= 180 <= extent[1]:
+        # in this case split into multilines
+        hem_west = lon < 0
+        # if all points on one side are +180 or -180, keep as a single line
+        if np.all(lon[hem_west] == -180) or np.all(lon[~hem_west] == 180):
```

Comment:
Seems like a strange condition, but of course an elegant solution if it's met!

Reply:
It's not equivalent though; it will definitely clean up those tracks that have orphaned points lying exactly on the "eastern half" of the dateline when moving westward. I agree that the `hem_west` should be replaced by a single condition for `hem_east`, of course.